src/logging.h

```diff
@@ -240,6 +230,17 @@ class Timer
 
 private:
     std::chrono::microseconds m_start_t{};
+
```

Comment:
In commit "logs: add timing information to FlushStateToDisk()" (07b8f8389c5ec557fbb6a7ab59a186865636d2d4)

Note: third commit is moving lines added by the second commit.

Reply:
Oops! Fixed, thanks.